### DISCUSSION TITLE
When I am train the yolov7 using train.py this error came could you help to solve the issue

### DISCUSSION THREAD

**Discussion by chintha-3** [roboflow/notebooks#38] [General]
![IMG_20230107_174513](https://user-images.githubusercontent.com/120492859/211153064-c8cfd3eb-0e2f-4cd9-acb8-087e62a75371.jpg)
![IMG_20230107_174513](https://user-images.githubusercontent.com/120492859/211153092-f4a2babb-3787-4f06-9598-d9e4004cc780.jpg)
Traceback (most recent call last):
  File "train.py", line 616, in <module>
    train(hyp, opt, device, tb_writer)
  File "train.py", line 363, in train
    loss, loss_items = compute_loss_ota(pred, targets.to(device), imgs)  # loss scaled by batch_size
  File "/content/drive/MyDrive/Final year project/yolov7/utils/loss.py", line 585, in __call__
    bs, as_, gjs, gis, targets, anchors = self.build_targets(p, targets, imgs)
  File "/content/drive/MyDrive/Final year project/yolov7/utils/loss.py", line 753, in build_targets
    _, cost_argmin = torch.min(cost[:, anchor_matching_gt > 1], dim=0)
RuntimeError: CUDA error: device-side assert triggered

**Comment by SkalskiP** [comment]
Hi, @chintha-3! 👋🏻 Thanks for moving that question from YouTube to GitHub. Is that error in our official notebook, or is that your custom code?

**Reply by chintha-3** [reply]
I train my custom dataset

**Reply by SkalskiP** [reply]
So our notebook but a custom dataset? Do I understand correctly? Could you share the link to your notebook?

**Reply by SkalskiP** [reply]
@chintha-3 ?

**Reply by chintha-3** [reply]
I tried with new custom dataset to train yolov7 with 50 images it trained successfully but when I detect the image it doesn't detect could you please help why this happens and how to solve it

**Reply by chintha-3** [reply]
> So our notebook but a custom dataset? Do I understand correctly? Could you share the link to your notebook? 

Actually when I labelling my images I accidentally saved one image in Pascal mode and I did all images in yolo.I thought this is the reason.I again did my custom data with 50 images and it trained properly but it doesn't detect could you please help me

**Reply by chintha-3** [reply]
Please give solution brother

**Reply by SkalskiP** [reply]
Can you zip the whole runs directory. And upload it here? I’d need to take a look at trading data to tell more.

**Reply by chintha-3** [reply]
runs zip file is very high how to upload it

**Reply by chintha-3** [reply]
> Can you zip the whole runs directory. And upload it here? I’d need to take a look at trading data to tell more. 

Can you suggest how to upload runs directory

**Reply by SkalskiP** [reply]
Hi 👋🏻 ! I was supper busy. Yo can:

1. zip it and upload it to your drive, and send me a link here
2. zip everything other than weights - those files weight the most and at the same time, are the least interesting to me
